pymeasure/adapters/telnet.py

```diff
@@ -33,6 +33,13 @@ class TelnetAdapter(Adapter):
     """ Adapter class for using the Python telnetlib package to allow
     communication to instruments
 
+    .. deprecated:: 0.11.2
+        Preparation for telnetlib deprecation.
```

Comment:
I'd say for the deprecated telnetlib package or such, that a user knows, that it is deprecated externally.